crates/evm/evm/src/executors/fuzz/mod.rs

```diff
@@ -146,8 +151,8 @@ impl FuzzedExecutor {
             success: run_result.is_ok(),
             reason: None,
             counterexample: None,
-            decoded_logs: decode_console_logs(&call.logs),
-            logs: call.logs,
+            decoded_logs: decode_console_logs(&logs.clone().into_inner()),
+            logs: logs.into_inner(),
```

Comment:
this clone can be avoided by tmp converting into_inner and borrow from there.

Reply:
> this clone can be avoided by tmp converting into_inner and borrow from there.

how about this? @mattsse